### ISSUE TITLE
Parinfer error with ignore token reader macro

### ISSUE THREAD

**Issue by reitzensteinm** [cursive-ide/cursive#1327]
This came up today - not sure if this is avoidable? Parinfer seems to be treating #_ as whitespace.

;Start with a case condition

```
(case foo
       1 2
       3 4
       5)

;Then we use the comment reader macro

#_(case foo
       1 2
       3 4
       5)

;Parinfer transforms it to this

#_(case foo)
   1 2
   3 4
   5
```

**Comment by cursive-ide** [comment]
Sadly, this case isn't really avoidable in general using vanilla parinfer. The same will happen if you have:
```
(def m {:a "a"
        :b "b"})
```
And then rename `m` to `my-map`:
```
(def my-map {:a "a"}
        :b "b")
```
I'm planning a hybrid mode which should hopefully fix this.

**Comment by cursive-ide** [comment]
This is fixed by the new parinfer smart mode (#1937).
